Automattic/wp-calypso
# composite-checkout: add missing cart events

#### Changes proposed in this Pull Request

* Adds `calypso_cart_product_add` and `calypso_cart_product_remove` events to composite checkout.

#### Testing instructions

Enter checkout, proceed to the review order step, and remove an item from your cart. Verify that the tracks events fire.

![image](https://user-images.githubusercontent.com/9310939/75581551-5852b700-5a2f-11ea-9f9b-6c72f1c6ef83.png)

![image](https://user-images.githubusercontent.com/9310939/75581115-85529a00-5a2e-11ea-9217-9e54ef841105.png)
